lib/actions/api-constants.ts

```diff
@@ -0,0 +1,5 @@
+export const RoutingQueryResult = {
```

Comment:
Would it be appropriate to rename to `ROUTING_QUERY_RESULT`? Even just to make it a bit more distinct from `routingQueryResult` in `batch-settings.tsx`

Reply:
Instead of changing the case, I picked a (better) name in https://github.com/opentripplanner/otp-react-redux/pull/990/commits/8a0e6c906d37de4c0a8359da6fde1f7d77544b44.